test/unit/search/fm_index/fm_index_collection_test_template.hpp

```diff
@@ -121,11 +121,11 @@ TYPED_TEST_P(fm_index_collection_test, concept_check)
 {
     using index_t = typename TypeParam::first_type;
 
-    EXPECT_TRUE((seqan3::fm_index_specialisation<index_t>));
+    EXPECT_TRUE((seqan3::detail::template_specialisation_of<index_t, seqan3::fm_index>));
```

Comment:
Wasn't sure if this concept_check should just be deleted like in the case of #2339